--- lsh-1.3.4/src/lsh_types.h.orig	Tue May 14 15:06:39 2002
+++ lsh-1.3.4/src/lsh_types.h	Tue May 14 15:07:41 2002
@@ -49,6 +49,18 @@
 # endif
 #endif
 
+#if SIZEOF_SHORT >= 8
+# define UINT64 unsigned short
+#elif SIZEOF_INT >= 8
+# define UINT64 unsigned int
+#elif SIZEOF_LONG >= 8
+# define UINT64 unsigned long
+#elif SIZEOF_LONG_LONG >= 8
+# define UINT64 unsigned long long
+#else
+# error No suitable type found to use for UINT64
+#endif /* UINT64 */
+ 
 #if SIZEOF_SHORT >= 4
 # define UINT32 unsigned short
 #elif SIZEOF_INT >= 4
