--- lsh-1.3.4/configure.in.orig	Tue May 14 13:49:32 2002
+++ lsh-1.3.4/configure.in	Tue May 14 13:49:39 2002
@@ -654,37 +654,6 @@
 
 # IPv6 support
 
-if test x$enable_ipv6 = xyes ; then
-  AC_CACHE_CHECK([if IPv6 is supported],
-    lsh_cv_sys_ipv6,
-    [AC_TRY_RUN([/* AF_INET6 avalable check */
-  #include <sys/types.h>
-  #include <sys/socket.h>
-  int main(int argc, char **argv)
-  {
-   if (socket(AF_INET6, SOCK_STREAM, 0) < 0)
-     exit(1);
-   else
-     exit(0);
-  }
-  ], lsh_cv_sys_ipv6=yes,
-     lsh_cv_sys_ipv6=no,
-     lsh_cv_sys_ipv6=no)])
-  
-  if test x$lsh_cv_sys_ipv6 != xyes ; then
-    enable_ipv6=no
-  elif test x$ac_cv_func_getaddrinfo != xyes ; then
-    AC_MSG_WARN([getaddrinfo not found. Disabling IPv6 support])
-    enable_ipv6=no
-  elif test x$ac_cv_func_gai_strerror != xyes ; then
-    AC_MSG_WARN([gai_strerror not found. Disabling IPv6 support])
-    enable_ipv6=no
-  elif test x$lsh_cv_sys_ai_numerichost != xyes ; then
-    AC_MSG_WARN([AI_NUMERICHOST not defined. Disabling IPv6 support])
-    enable_ipv6=no
-  fi
-fi
-
 AH_TEMPLATE([WITH_IPV6], [For Ipv6 support])
 if test x$enable_ipv6 = xyes ; then
   AC_DEFINE(WITH_IPV6)
