diff -urNp -x '*.orig' lsh-2.1.org/doc/lsh.texinfo lsh-2.1/doc/lsh.texinfo
--- lsh-2.1.org/doc/lsh.texinfo	2013-03-09 08:49:46.000000000 +0100
+++ lsh-2.1/doc/lsh.texinfo	2023-02-02 08:54:55.593167326 +0100
@@ -7,9 +7,9 @@
 
 @documentencoding ISO-8859-1
 
-@dircategory GNU Packages
+@dircategory Net utilities:
 @direntry
-* LSH: (lsh).           Secure Shell and related utilities.
+* LSH: (lsh).				Secure Shell and related utilities
 @end direntry
 
 @set UPDATED-FOR 2.0
